### PR TITLE
Remove db access code

### PR DESCRIPTION
**Pull request by mkurc-ant**
This PR removes all database access code and changes all imports to use the `prjuray-tools` repo for that purpose.

### CODE REVIEW

**Review by litghost — CHANGES_REQUESTED** [review]
Looks like we are missing some .gitignore for the prjuray-tools submodule.  I would assume lines like: https://github.com/SymbiFlow/prjuray/blob/53f3c94b58ffc6d405ac20a3b340ae726717ed47/.gitignore#L14-L15

**Comment by mkurc-ant** [comment]
@litghost There is an open PR for that: https://github.com/SymbiFlow/prjuray-tools/pull/5

File: Makefile [hunk 1]
```diff
@@ -24,10 +24,15 @@ env:
 	virtualenv --python=python3 env
 	# Install utils
 	ln -sf $(PWD)/utils env/lib/python3.*/site-packages/
+	# Install prjuray-tools
+	ln -sf $(PWD)/third_party/prjuray-tools/prjuray env/lib/python3.*/site-packages/
```
Comment (litghost) [inline Makefile:28]: This should be added to `requirements.txt`, which will on the next line will be installed in setup mode with a sym link.

File: utils/environment.python.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@
 #
 # SPDX-License-Identifier: ISC
 # FIXME: fasm should be installed into the running Python environment.
-export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:$PYTHONPATH"
+export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:${URAY_DIR}/third_party/prjuray-tools:$PYTHONPATH"
```
Comment (litghost) [inline utils/environment.python.sh:11]: This entire line is likely no longer required, as the venv should already have fasm and prjuray-tools?  I think this is leftover techdebt from before the venv was created?

File: utils/environment.sh [hunk 3]
```diff
@@ -28,7 +28,7 @@ fi
 
 # misc
 export URAY_PART_YAML="${URAY_DATABASE_DIR}/${URAY_DATABASE}/${URAY_PART}/part.yaml"
-export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:$PYTHONPATH"
+export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:${URAY_DIR}/third_party/prjuray-tools:$PYTHONPATH"
```
Comment (litghost) [inline utils/environment.sh:31]: This entire line is likely no longer required, as the venv should already have fasm and prjuray-tools? I think this is leftover techdebt from before the venv was created?

**Review by litghost — CHANGES_REQUESTED** [review]
Comments in

File: Makefile (hunk 1 above)
Comment (mkurc-ant) [reply Makefile:28]: Ok. But for that I have to add `setup.py` to `prjuray-tools/prjuray`.

Comment (mkurc-ant) [reply Makefile:28]: I've opened a PR with that: https://github.com/SymbiFlow/prjuray-tools/pull/6

File: utils/environment.python.sh (hunk 2 above)
Comment (mkurc-ant) [reply utils/environment.python.sh]: That's correct. Fixed.

File: utils/environment.sh (hunk 3 above)
Comment (mkurc-ant) [reply utils/environment.sh]: Same here. Fixed.

File: Makefile (hunk 1 above)
Comment (mkurc-ant) [reply Makefile:28]: The PR had been merged.

Comment (litghost) [reply Makefile:28]: This line should be removed now?

**Review by litghost — CHANGES_REQUESTED** [review]
Still need to remove site-package symlink?

**Comment by mkurc-ant** [comment]
@litghost I've removed the leftover symlink.

**Pull request merged**